ocs_ci/ocs/machine.py

```diff
@@ -973,3 +973,50 @@ def add_annotation_to_machine(annotation, machine_name):
     command = f"annotate machine {machine_name} {annotation}"
     log.info(f"Adding annotation: {command} to machine {machine_name} ")
     ocp_obj.exec_oc_cmd(command)
+
+
+def set_replica_count(machine_set, count):
+    """
+    Change the replica count of a machine set.
+
+    Args:
+          machine_set (str): Name of the machine set
+          count (int): The number of the new replica count
+
+    Returns:
+        bool: True if the change was made successfully. False otherwise
+
+    """
+    ocp = OCP(namespace=constants.OPENSHIFT_MACHINE_API_NAMESPACE)
+    ocp.exec_oc_cmd(f"scale --replicas={count} machinesets {machine_set}")
+    return True
```

Comment:
it always return True?

Reply:
TBH I took it from this function here https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/machine.py#L903. I think the return value is just an indication that the function was executed successfully. Maybe we can add `try` and `exception` to catch an error and return also the `False` value.